jetty/solr/blacklight-core/conf/solrconfig.xml

```diff
@@ -222,6 +222,7 @@
          artwork_subtype,
```

Comment:
Hi. We don't use the stuff in the jetty folder anymore. It's been deprecated since that belongs to Solr 4. We've migrated in april to Solr 7 due to dependency issues in Ruby.  The new configuration now resides in the solr-conf folder.

Are the commits in the files in the jetty folder perhaps older stuff that was part of this PR?